src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -371,17 +371,7 @@ trait ScalaSettings extends AbsScalaSettings
   val YoptLogInline = StringSetting("-Yopt-log-inline", "package/Class.method", "Print a summary of inliner activity; `_` to print all, prefix match to select.", "")
 
   import scala.reflect.internal.util.Statistics
-  object YstatisticsPhases extends MultiChoiceEnumeration { val parser, typer, patmat, erasure, cleanup, jvm = Value }
-  val Ystatistics = {
-    val description = "Print compiler statistics for specific phases"
-    MultiChoiceSetting(
-      name    = "-Ystatistics",
-      helpArg = "phase",
-      descr   = description,
-      domain  = YstatisticsPhases,
-      default = Some(List("_"))
-    )
-  }
```

Comment:
I need to tweak this to set the default back to "parser,typer,patmat,erasure,cleanup,jvm"

Reply:
Is this comment a `TODO`? Looks to me like you've done it already.